tests/acceptance/settings-test.js

```diff
@@ -8,13 +8,13 @@ import { setupApplicationTest } from 'cargo/tests/helpers';
 
 import axeConfig from '../axe-config';
 
-module('Acceptance | /crates/:name/owners', function (hooks) {
+module('Acceptance | /crates/:name/settings', function (hooks) {
```

Comment:
can you add a test that checks that the redirection from `/crates/:name/owners` to `/crates/:name/settings` works correctly? 🙏

Reply:
Added.